snapcraft/internal/meta/_errors.py

```diff
@@ -21,3 +21,72 @@ class CommandError(errors.SnapcraftError):
 
     def __init__(self, message: str) -> None:
         self.fmt = message
+
+
+class SnapMetaGenerationError(errors.SnapcraftError):
+    pass
+
+
+class SnapcraftYamlMissingRequiredValue(SnapMetaGenerationError):
```

Comment:
Just looking at the errors, I can't figure out the difference between this error class and `MissingSnapcraftYamlKeys`.

Reply:
After reading where it's used, it sounds like this should be part-specific, no?